.changeset/long-beds-mate.md

```diff
@@ -0,0 +1,27 @@
+---
+'gatsby-plugin-clerk': major
+'@clerk/localizations': major
+'@clerk/nextjs': major
+'@clerk/clerk-react': major
+'@clerk/remix': major
+'@clerk/types': major
+---
+
+Drop deprecations. Migration steps:
+- drop `orgs` jwt claim from session token
+- change type of `auth` param of `withServerAuth()` callback to `AuthObject` from `ServerSideAuth` in `gatsby-clerk-plugin`
+    - use `auth.sessionClaims` instead of `auth.claims`
+    - use `AuthObject` properties from `auth`
+- use `publishableKey` instead of `frontendApi`
+- drop `formFieldLabel__emailAddress_phoneNumber` from localization keys
+- drop `formFieldLabel__phoneNumber_username` from localization keys
+- drop `formFieldLabel__emailAddress_phoneNumber_username` from localization keys
+- drop `formFieldInputPlaceholder__emailAddress_phoneNumber` from localization keys
```

Comment:
Are all these replaced with anything else or are they dropped completely?

Reply:
They are completed dropped. If we have a replacement i refer to it by using the following syntax:
```
- use `{replacement}` instead of `{deprecated}`
```